src/open_inwoner/openklant/tests/factories.py

```diff
@@ -1,9 +1,37 @@
 import factory
 
+from open_inwoner.openklant.api_models import ContactMoment
+
 
 class ContactFormSubjectFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = "openklant.ContactFormSubject"
 
     subject = factory.Faker("sentence")
     subject_code = factory.Faker("word")
+
+
+def make_contactmoment(contact_moment_data: dict):
```

Comment:
I think instead of this, we could use from `zgw_consumers.api_models.base.factory`:

```python
from zgw_consumers.api_models.base import factory

factory(ContactMoment, data)
```